refinery/core/models.py

```diff
@@ -1379,12 +1379,15 @@ def get_input_file_uuid_list(self):
             input_file_uuid_list.append(cur_fs_uuid)
         return input_file_uuid_list
 
-    def data_sets_query(self):
-        analysis_facet_name = '{}_{}_{}_s'.format(
+    def facet_name(self):
+        return '{}_{}_{}_s'.format(
             NodeIndex.ANALYSIS_UUID_PREFIX,
             self.data_set.get_latest_study().id,
             self.data_set.get_latest_assay().id,
         )
+
+    def data_sets_query(self):
```

Comment:
Do we even want to keep this around at all? Is it ever the right way of doing what it's trying to do?